core/src/functions/content.functions.ts

```diff
@@ -31,7 +31,8 @@ export const convertBibleInputToBookPlaintext = (
     contents: IBibleContentForInput[],
     _accChapters: BibleBookPlaintext = new Map()
 ) => {
-    for (const content of contents) {
+    const validContents = contents.filter(content => content);
```

Comment:
see similar issue above. if a check is really needed, it should throw an error within the for loop imho